.azure/pipelines/ci.yml

```diff
@@ -30,507 +36,534 @@ variables:
               -sign
-              /bl:artifacts/log/build.codesign.binlog
+              -buildInstallers
+              /bl:artifacts/log/installers.msbuild.binlog
               $(_BuildArgs)
-      displayName: Code sign packages
+        displayName: Build Installers
 
-    # Windows installers bundle both x86 and x64 assets
-    - script: ./build.cmd
-            -ci
-            -sign
-            -buildInstallers
-            /bl:artifacts/log/installers.msbuild.binlog
-            $(_BuildArgs)
-      displayName: Build Installers
+      artifacts:
+      - name: Windows_Logs
+        path: artifacts/log/
+        publishOnError: true
+      - name: Windows_Packages
+        path: artifacts/packages/
+      - name: Windows_Symbols
+        path: artifacts/symbols/
+      - name: Windows_VSIX
+        path: artifacts/VSSetup/
+      - name: Windows_Manifests
+        path: artifacts/manifests/
+      - name: Windows_Installers
+        path: artifacts/installers/
 
-    artifacts:
-    - name: Windows_Logs
-      path: artifacts/log/
-      publishOnError: true
-    - name: Windows_Packages
-      path: artifacts/packages/
-    - name: Windows_Symbols
-      path: artifacts/symbols/
-    - name: Windows_VSIX
-      path: artifacts/VSSetup/
-    - name: Windows_Manifests
-      path: artifacts/manifests/
-    - name: Windows_Installers
-      path: artifacts/installers/
+  # Build Windows ARM
+  - template: jobs/default-build.yml
+    parameters:
+      codeSign: true
+      jobName: Windows_arm_build
+      jobDisplayName: "Build: Windows ARM"
+      agentOs: Windows
+      buildArgs:
+        -arch arm
+        -sign
+        -pack
+        -noBuildNodeJS
+        -noBuildJava
+        /p:OnlyPackPlatformSpecificPackages=true
+        /bl:artifacts/log/build.win-arm.binlog
+        $(_BuildArgs)
+      installNodeJs: false
+      installJdk: false
+      artifacts:
+      - name: Windows_arm_Logs
+        path: artifacts/log/
+        publishOnError: true
+      - name: Windows_arm_Packages
+        path: artifacts/packages/
+      - name: Windows_arm_Manifests
+        path: artifacts/manifests/
+      - name: Windows_arm_Installers
+        path: artifacts/installers/
 
-# Build Windows ARM
-- template: jobs/default-build.yml
-  parameters:
-    codeSign: true
-    jobName: Windows_arm_build
-    jobDisplayName: "Build: Windows ARM"
-    agentOs: Windows
-    buildArgs:
-      -arch arm
-      -sign
-      -pack
-      -noBuildNodeJS
-      -noBuildJava
-      /p:OnlyPackPlatformSpecificPackages=true
-      /bl:artifacts/log/build.win-arm.binlog
-      $(_BuildArgs)
-    installNodeJs: false
-    installJdk: false
-    artifacts:
-    - name: Windows_arm_Packages
-      path: artifacts/packages/
-    - name: Windows_arm_Manifests
-      path: artifacts/manifests/
-    - name: Windows_arm_Installers
-      path: artifacts/installers/
-    - name: Windows_arm_Logs
-      path: artifacts/log/
-      publishOnError: true
```

Comment:
Moving these was a nit. Best to get the logs uploaded first in case of a cancelation.